components/FilesTable/OsparcFileViewersDialog.vue

```diff
@@ -0,0 +1,105 @@
+<template>
```

Comment:
Formatting is off, which should have been caught by the linter. I'm making a note to check into the linting checks. Checking `yarn lint` shows quite a few linting errors in the code base, so something is not right.